src/config.rs

```diff
@@ -190,6 +199,8 @@ impl Default for TermvizConfig {
                 (input::ZOOM_OUT.to_string(), "-".to_string()),
                 (input::INCREMENT_STEP.to_string(), "k".to_string()),
                 (input::DECREMENT_STEP.to_string(), "j".to_string()),
+                (input::NEXT.to_string(), "n".to_string()),
+                (input::PREVIOUS.to_string(), "b".to_string()),
```

Comment:
previous should be N to be compliant with superior vim keymappings

Reply:
do you want m for next and n for previous?

Reply:
on vim n is next match, N is previous match for searches